### PR TITLE
Possibility to suppress bootstrap on some bookmarks

### PR DESCRIPTION
**Pull request by krystian8207**
### How to test:
Run [example](https://github.com/Appsilon/shiny.router/blob/a2936ee45a68ba41863dab19112ddc3f5e33e6d4/examples/disable_bootstrap.R)

### CODE REVIEW

File: R/pages.R [hunk 1]
```diff
@@ -25,3 +25,31 @@ page404 <- function(page = NULL, message404 = NULL){
     return(page)
   }
 }
+
+#' Fix conflicts when some bookmark uses bootstrap
+#'
+#' @description This function dynamically removes bootstrap dependency when user opens specified bookmark.
```
Comment (dokato) [inline R/pages.R]: I think that `@description` here is redundant as everything below the title and without tag is taken as a description.

File: R/pages.R [hunk 2]
```diff
@@ -25,3 +25,31 @@ page404 <- function(page = NULL, message404 = NULL){
...
+#' It should be inserted in head of bootrstrap page.
+#' @param bookmark Bookmark name on which bootstrap dependency should be suppressed.
+#'
+#' @export
+disable_bootstrap_on_bookmark <- function(bookmark) {
+  bootstrap_dependency <- htmltools::renderDependencies(list(shiny:::bootstrapLib()), srcType = "href")
+  tagList(
```
Comment (dokato) [inline R/pages.R]: Please add `@import shiny` to docs.

Comment (krystian8207) [reply R/pages.R]: Based on: http://r-pkgs.had.co.nz/namespace.html I think we should make some code cleaning:
- remove `shiny::` across all functions
- add `@import shiny` to our namespace
But this should be included in separate PR. 
WDYT?

Comment (dokato) [reply R/pages.R]: Ok, so please just create an issue on that.

**Pull request merged**